getpaid/backends/pagseguro/tests.py

```diff
@@ -0,0 +1,16 @@
+"""
+This file demonstrates writing tests using the unittest module. These will pass
+when you run "manage.py test".
+
+Replace this with more appropriate tests for your application.
+"""
+
+from django.test import TestCase
+
+
+class SimpleTest(TestCase):
+    def test_basic_addition(self):
+        """
+        Tests that 1 + 1 always equals 2.
+        """
+        self.assertEqual(1 + 1, 2)
```

Comment:
Another question, I wasted time trying setup the tests on my PC. I used the some version of pip on TravisCI, executed the some script, but didn’t worked very well. Maybe, it would be more easy.